examples/experimental/trial/gaea_nas/const.yaml

```diff
@@ -0,0 +1,40 @@
+description: gaea_nas
+
+data:
+  download_dir: /data
+
+bind_mounts:
+  - host_path: /tmp
+    container_path: /data
+    read_only: false
+
+hyperparameters:
+    global_batch_size: 256
+    learning_rate: 0.1
+    momentum: 0.9
+    min_learning_rate: 0
+    scheduler_epochs: 50
+    weight_decay: 3e-4
+    arch_learning_rate: 0.1
+    init_channels: 16
+    layers: 8
+    nodes: 4
+
+resources:
+  slots_per_trial: 2
```

Comment:
Since this is const.yaml, we don't have slots_per_trial to be more than 1 here?

Reply:
i'm not a fan of differentiating between const and distributed.  const to me means constant hps instead of a search space.